starknet-rpc-test/contracts/HelloStarknet.cairo

```diff
@@ -0,0 +1,19 @@
+#[starknet::contract]
+mod HelloStarknet {
```

Comment:
Not a gib fan of this contract name and file name.
Can you try something more descriptive: `Counter` or `Balance`

Reply:
fixed